apps/src/music/FieldSounds.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import SoundsPanel from './SoundsPanel';
+import GoogleBlockly from 'blockly/core';
+
+class FieldSounds extends GoogleBlockly.Field {
```

Comment:
For our other labs, these are usually defined in `apps/srcs/blockly/addons` and the filenames are preprended with `cdo` (e.g. `cdoFieldDropdown.js`) to clarify that this is one of our custom field. I think the plan is to worry about some of these final decisions a little later on?

Reply:
That's a good point.  I think we could hold off on this for a bit longer.  The implementation is quite specific to Music Lab right now.